<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일(F)</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>편집(E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
    <addaction name="action_next_find"/>
    <addaction name="action_prev_find"/>
    <addaction name="action_find_replace"/>
    <addaction name="action_go"/>
    <addaction name="separator"/>
    <addaction name="action_all_select"/>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menu_form">
    <property name="title">
     <string>서식(O)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>보기(V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>도움말</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_form"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>열기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCtrl_O">
   <property name="text">
    <string>Ctrl + O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>끝내기</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>실행 취소</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>잘라내기</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>복사</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>붙여넣기</string>
   </property>
  </action>
  <action name="action_find">
   <property name="text">
    <string>찾기 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_next_find">
   <property name="text">
    <string>다음 찾기</string>
   </property>
  </action>
  <action name="action_prev_find">
   <property name="text">
    <string>이전 찾기</string>
   </property>
  </action>
  <action name="action_find_replace">
   <property name="text">
    <string>바꾸기</string>
   </property>
  </action>
  <action name="action_go">
   <property name="text">
    <string>이동</string>
   </property>
  </action>
  <action name="action_all_select">
   <property name="text">
    <string>모두 선택</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>시간/날짜</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
